docs/schemas/advanced.md

```diff
@@ -53,13 +93,14 @@ enums:
       UNDEAD:
 ```
 
-Note that these constructs may ignored by some generators in the linkml 1.1 series.
+Note that the range of `vital_status` is declared as `Any`, which is further constrained by the `any_of` expression.
+
+Currently, it is important to have this base range declaration, because LinkML constraint semantics are
+monotonic. If this were not stated, then the `default_range` of string would be applied.
```

Comment:
"monotonic" may be unclear for non-experts. Having an explanation would be helpful. I did not know about the non-math meaning of "monotonic" before you used it in the discussions some time ago.

Reply:
Thx! Linked back to slots docs for this, and added a bit of qualification